src/scripts/main.js

```diff
@@ -1,3 +1,27 @@
 'use strict';
 
-// write code here
+const thead = document.querySelector('thead');
+const tbody = document.querySelector('tbody');
+const rows = [...tbody.querySelectorAll('tr')];
+
+thead.addEventListener('click', e => {
+  const columnIndex = e.target.cellIndex;
+
+  rows.sort((a, b) => {
+    const current = a.cells[columnIndex].innerHTML;
+    const next = b.cells[columnIndex].innerHTML;
+
+    if (e.target.cellIndex === 0 || e.target.cellIndex === 1) {
+      return current.localeCompare(next);
+    } else if (e.target.cellIndex === 2) {
+      return Number(current) - Number(next);
+    } else if (e.target.cellIndex === 3) {
+      return salaryToNumber(current) - salaryToNumber(next);
+    }
```

Comment:
This could be writen witoud else if
```suggestion

if (columnIndex === 2) {
  return Number(current) - Number(next);
}

if (columnIndex === 3) {
  return salaryToNumber(current) - salaryToNumber(next);
}

return current.localeCompare(next);
```